src/components/pacbio/PacbioLibraryCreate.vue

```diff
@@ -125,26 +135,57 @@ export default {
   },
   data() {
     return {
-      library: { tag: { group_id: '' }, sample: {} },
-      tagOptions: [{ value: '', text: 'No tag' }],
+      library: { tag: { id: '' }, sample: {} },
+      selectedTagSetId: '',
+      selectedTagId: '',
     }
   },
   computed: {
-    ...mapGetters('traction', ['tractionTags']),
+    // Map fetched Pacbio tag sets and tags
+    ...mapState('traction/pacbio/poolCreate', {
+      tagSets: (state) => state.resources.tagSets,
+      tags: (state) => state.resources.tags,
+    }),
+    // Return options for the first dropdown
+    tagSetOptions() {
+      const tagSets = Object.values(this.tagSets)
```

Comment:
I would use a getter to get TagSets/tags instead of map state in the component.

Reply:
I have replaced mapState with mapGetters to get options from the state.